src/gps.pod

```diff
@@ -0,0 +1,71 @@
+=encoding UTF-8
+
+=head1 NAME
+
+gps - Documentation of collectd's C<gps plugin>
+
+=head1 SYNOPSIS
+
+  # See collectd.conf(5)
+  LoadPlugin gps
+  <Plugin "gps">
+    Host "localhost"
+    Port "2947"
+    Timeout 1000
+  </Plugin>
+
+=head1 DESCRIPTION
+
+The C<gps plugin> connects to gpsd on the host machine.
+The port and the timeout are configurable. 
+
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<Host>
+
+The host on which gpsd runs.
+
+=item B<Port>
+
+Port to connect to gpsd on the host machine.
+
+=item B<Timeout>
+
+Timeout in ms.
```

Comment:
We're trying to default to seconds, rather than having a mix of seconds, milliseconds, microseconds, … in config options. Ideally, you'd use the type `cdtime_t` to store the value in your code.